libs/partners/cohere/tests/integration_tests/test_chat_models.py

```diff
@@ -121,7 +121,7 @@ class Person(BaseModel):
     }
 
     # where it doesn't call the tool
-    strm = tool_llm.stream("What is 2+2?")
+    strm = llm.stream("What is 2+2?")
```

Comment:
This was right before! Does cohere's tool calling always force a tool call, even if the query isn't relevant to the provided tool?

Reply:
The default behaviour is to indicate that the provided tools are irrelevant. In these cases an empty tool_calls is returned. I can fix the implementation so that it doesn't populate kwargs.tool_calls in this case!